openassessment/templates/openassessmentblock/edit/oa_edit_rubric.html

```diff
@@ -38,7 +38,7 @@
                     <textarea id="openassessment_rubric_feedback" class="input setting-input">{{ feedbackprompt }}</textarea>
                 </div>
                 <p class="setting-help">
-                    {% trans "Encourage your students to provide feedback on the response they've graded. You can replace the sample text with your own." %}
+                    {% trans "Encourage your learners to provide feedback on the response they have graded. You can replace the sample text with your own." %}
```

Comment:
Nit: I find the phrase "Encourage your learners..." to be a little strange. It works for me as "Encourage your students..." but I don't know that the authors feel that they own the learners in a course. :-) FYI @catong